src/modules/CC1101/CC1101.cpp

```diff
@@ -489,6 +499,25 @@ int16_t CC1101::setRxBandwidth(float rxBw) {
   return(RADIOLIB_ERR_INVALID_RX_BANDWIDTH);
 }
 
+int16_t CC1101::CalcRxBandwidth() {
+    // Uncertainty ~ +/- 40ppm for a cheap CC1101
+    // Uncertainty * 2 for both transmitter and receiver
+    float uncertainty = (getFrequency() * 40 * 2);
+    uncertainty = (uncertainty/1000); //Since bitrate is in kBit
+    float minbw = (getBitRate() + uncertainty);
+    
+    int possibles[16] = {58, 68, 81, 102, 116, 135, 162, 203, 232, 270, 325, 406, 464, 541, 650, 812};
+    
+    for (int i = 0; i < 16; i++) {
+      if (possibles[i] > minbw) {
+        int16_t state = setRxBandwidth(possibles[i]);
+        return(state);
+        break;
+      }
+    }
+    
```

Comment:
Return value from this method is missing, it must return something (e.g. `RADIOLIB_ERR_UNKNOWN` or some "invalid bandwidth" error code if it never breaks out of the loop.

Reply:
I added a return value, but that return value will never be reached (why I didn't add it initially). You have checks for both Freq and bitrate and the only way to get through the loop without a bw being selected is if you are well outside the hw scope. 

I made that mistake years ago with my Yardstick one, accidentally set it to 43.392 instead of 433.92

Reply:
Yes, if everything works correctly, the for loop will return. However, if for whatever reason the configured bandwidth is nonsense, the loop may not return, ehnce why the final return is actually needed, so thank you for adding it.

ESP32 build has enabled -Werror, the missing return creates a warning that a non-void function may not return a value, and that gets treated like an error due to the -Werror switch - see the CI run: https://github.com/jgromes/RadioLib/actions/runs/8451714866/job/23150617366#step:6:1237